locations/templates/locations/edit.html

```diff
@@ -7,12 +7,33 @@
             <form method="POST" enctype="multipart/form-data">
                 {% csrf_token %}
                 {{ form|bulma }}
-                <button type="submit" class="button is-primary">Save Scene</button>
+                <p><strong>Race Distribution / Occupancy:</strong></p>
+                <div id="form_set">
+                    {{ occupancies.management_form }}
+                    {% for extra_form in occupancies.forms %}
+                        {{ extra_form.non_field_errors }}
+                        {{ extra_form.errors }}
+                        <div class="no_error">
+                            {{ extra_form|bulma }}
+                        </div>
+                    {% endfor %}
+                </div>
+                <input type="button" value="Additional Inhabitants" id="add_more" class="button is-info">
+                <div id="empty_form" style="display:none">
+                    <table class='no_error'>
+                        {{ occupancies.empty_form|bulma }}
+                    </table>
+                </div>
+                <hr/>
+                <button type="submit" class="button is-primary">Update Location</button>
             </form>
         </div>
     </div>
 {% endblock %}
 
 {% block extra_media %}
     {{ form.media }}
+    <script>
+        activate_dynamic_inline_forms("racedistribution");
```

Comment:
This should also be wrapped in `$(document).ready();`
```suggestion
        $(document).ready(function() {
            activate_dynamic_inline_forms("racedistribution");
        });
```